task_2.py

```diff
@@ -1,2 +1,7 @@
-def chess_board(n, k):
-    pass
\ No newline at end of file
+def chess_board(n,k):
+    for p in range(0,2 * n):
```

Comment:
Jak nie będziesz używać `p` to lepiej wstawić w to miejsce `_`. Wewnątrz `range` argumenty oznaczają początek i koniec (jeśli są podane dwa). Bazowo początek ma wartość 0, więc w tym przypadku wystarczyłoby napisać `for _ in range(2 * n)`. Zauważ, że `2 * n` jest tutaj nieprawidłową wartością, bo Twoja szachownica jest 2 razy dłuższa niż powinna. Ostatecznie powinno być zatem `for _ in range(n)`.